exp-player/addon/components/exp-exit-survey.js

```diff
@@ -148,7 +147,7 @@ export default ExpFrameBaseComponent.extend({
                 Ember.merge(formData, this.getValue());
                 root.set('formData', formData);
                 console.log('Post-study survey complete.');
-                root.actions.next();
+                root.actions.next.apply(root);
```

Comment:
I was having difficulty getting this to work without the strategy used here. Not sure why, but the action seemed to be skipping the exp-frame-base action.

If you can confirm a solution that is more Ember-friendly I'd be happy to change this, but I did spend ~20-30 min trying to figure this out and hesitate to invest too much more time.